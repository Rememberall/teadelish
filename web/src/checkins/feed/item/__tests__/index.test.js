import React from 'react';
import renderer from 'react-test-renderer';

import Item from '../index';

it('should not change', () => {
  const tree = renderer
    .create(
      <Item
        beverage={{
          brand: 'Illy',
          name: 'Espresso'
        }}
        rating={5}
        comment="A rather short comment"
      />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
