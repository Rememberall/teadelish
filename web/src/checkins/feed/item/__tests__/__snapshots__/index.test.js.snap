// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should not change 1`] = `
<li
  className="checkin-feed-item"
>
  <img
    alt=""
    src="http://www.swedishfoodshop.com/media/catalog/product/cache/1/image/9df78eab33525d08d6e5fb8d27136e95/r/u/russian_earl_grey_20-pack_40g.jpg"
  />
  <div>
    <h1>
      Illy
       
      Espresso
    </h1>
    <p>
      5
       / 5
    </p>
    <p>
      A rather short comment
    </p>
  </div>
</li>
`;
